README.md

```diff
@@ -30,9 +30,14 @@ $ curl -u joel:joel1234 -sS http://localhost:5000/users -X GET
 Example from AngularJS:
 
 ```
+import { Http, Response, Headers } from '@angular/http'
+
```

Comment:
This doesn't seem to be proper markdown, right? See https://github.com/EyeSeeTea/FIRE-MockServer/blob/4e3d972b2bc001272281acc47713f71eeb038527/README.md